Makefile

```diff
@@ -89,9 +89,3 @@ server/openapi: $(OPENAPI_SRC_FILES)
 	rm -rf $(BASE)/tmp
 
 	cd $(BASE); go fmt ./...
-
```

Comment:
Not related to this PR, but I wanted to remove it to avoid accumulating clutter on our codebase.